cpu/k60/Makefile.features

```diff
@@ -0,0 +1 @@
+FEATURES_PROVIDED += periph_pm
```

Comment:
I's say it makes sense to move this to `cpu/kinetis_common/Makefile.features`, as all the kinetis CPUs share the PM stuff...